src/components/svgs/signals.js

```diff
@@ -0,0 +1,38 @@
+//TODO: this a temporary way to display these svg
+import React from 'react'
```

Comment:
lets convert this to use img tag instead

Reply:
Alright